scripts/translation/README.md

```diff
@@ -62,15 +81,20 @@ php doc-base/scripts/translation/qaxml.t.php function
 php doc-base/scripts/translation/qaxml.t.php interfacename
 php doc-base/scripts/translation/qaxml.t.php parameter
 php doc-base/scripts/translation/qaxml.t.php type
-
 php doc-base/scripts/translation/qaxml.t.php classsynopsis
 php doc-base/scripts/translation/qaxml.t.php constructorsynopsis
 php doc-base/scripts/translation/qaxml.t.php destructorsynopsis
 php doc-base/scripts/translation/qaxml.t.php fieldsynopsis
 php doc-base/scripts/translation/qaxml.t.php funcsynopsis
 php doc-base/scripts/translation/qaxml.t.php methodsynopsis
+
+php doc-base/scripts/translation/qaxml.t.php code
```

Comment:
Note: the code, literal, filename and varname are  in both the expected no translation and maybe translation categories.

Reply:
Fixed. These categories may fluctuate between translations, or even in different partis inside a translation.